packages/encryption/src/image-steganography/getDimensionAsBlob.ts

```diff
@@ -0,0 +1,34 @@
+import type { Dimension } from './presets.js'
+
+export async function getDimensionAsBlob(file: Blob | string): Promise<Dimension | undefined> {
```

Comment:
I don't think this should go here. Maybe you should add it in https://github.com/DimensionDev/kit this library (`@masknet/kit`). If you need help, please ping me

Reply:
please remove this file as it has already landed in the `kit` package.